modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-filtering-ebpf-rule_{context}"]
+= Filtering eBPF flow data using a global rule
+You can configure the `FlowCollector` to filter eBPF flows using a global rule as a method to control the flow of packets cached in eBPF's flows table based on certain configuration.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A few sample configurations are as follows:
++
+
+[%collapsible]
+.Filter Kubernetes service traffic to specific POD IP endpoint
```

Comment:
```suggestion
.Filter Kubernetes service traffic to a specific POD IP endpoint
```

Reply:
Yep and idk why I allcaps POD here but I think it should be Pod.